### DISCUSSION TITLE
Upscaling.

### DISCUSSION THREAD

**Discussion by williamsblood** [lllyasviel/Fooocus#800] [Q&A]
So, when I use upscaling, should I keep the positive prompt or should I delete the field? I've noticed that the upscaled image sometimes changes significantly when the prompt is present.

**Comment by lllyasviel** [comment]
yes keep the prompt
image will vary so that bad face will be fixed
use Upscale (Fast 2x) if you do not want any change

if you know what you are doing, the developer tools has an option `Forced Overwrite of Denoising Strength of "Upscale"`
